_docs/guides/best-practices.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Best Practices 
+order: 004
+---
+# {{ page.title }}
+
+## Service Accounts
+When you use multiple service accounts with your Forseti Security deployments,
+you fulfill the security best practice of least privilege. Following are the
+scenarios for which it's best to use separate service accounts:
+
+ * **[Forseti Security Service Account](#forseti-security-service-account)**
+ (required): Used by all core modules of the program to provide basic
+ inventory, scanning, and enforcement actions.
+ * **[GSuite Groups Service Account](#gsuite-groups-service-account)**
+ (optional): Used to inventory GSuite Groups and their members.
+ Forseti IAM explain requires this to be enabled.
+ * **[Forseti Explain Service Account](#forseti-explain-service-account)**
+ (optional): Used to provide IAM explain functionality.
+
+When you name service accounts, it's best to use a descriptive name like
+`forseti-security` or `forseti-security-gsuite`.
+
+### Forseti Security Service Account
+This service account is used by core modules of the Forseti service. For
+example, `forseti_inventory` uses this service account to read and store the
+supported resources. It's also used by `forseti_scanner` to scan
+`forseti_inventory` records.
+
+A good name for this service account would be `forseti-security`.
+
+### GSuite Groups Service Account
+It's best to use a separate service account for GSuite Groups inventory for
+privilege separation because the service account key must be local to
+`forseti_inventory`. By using a separate service account, the key scope is
+limited to GSuite Groups if the machine is compromised.
+
+If you [enable]{% link _docs/howto/gsuite-group-collection.md %})
+GSuite Group inventory and create a service account, a good name
+for the service account would be `forseti-security-gsuite-dwd`.
```

Comment:
To my point and per the discussion above I think it would be better to s/dwd/groups/